contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -333,7 +333,9 @@ contract VaultAdmin is VaultStorage {
      */
     function priceUSDMint(address asset) external view returns (uint256) {
         uint256 price = IOracle(priceProvider).price(asset);
-        if (price > 1e8) {
+        if (price < MINT_MINIMUM_ORACLE) {
```

Comment:
Am I correct in thinking this only gets used in generating estimates of how much OUSD a user will receive in return for their stablecoins? Is there a reason to return 0 instead of reverting here? If it's for displaying in the UI it seems like it'd be clearer to make the error explicit rather than returning something that is still a potentially valid value (0).

Reply:
+1 for returning an error so that the UI can handle it appropriately.